using-mobile-extensions/adobe-target/README.md

```diff
@@ -317,6 +317,26 @@ Target.setPreviewRestartDeepLink("myApp://HomePage");
 {% endtab %}
 {% endtabs %}
 
+The `collectLaunchInfo` API is used to enter visual preview mode. After visual preview mode is enabled, a red colored floating button is displayed on the app screen which can be pressed to easily re-enter visual preview mode.
```

Comment:
- Please add "the" between "enter" and "visual preview mode".

- Please change the second sentence so that it reads as follows:
"After the visual preview mode is enabled, a red floating button is displayed on the app screen. This button can be pressed to enter the visual preview mode again."